recycler/src/main/java/com/elpassion/android/commons/recycler/strategies/id/ItemIdStrategy.kt

```diff
@@ -0,0 +1,5 @@
+package com.elpassion.android.commons.recycler.strategies.id
+
+interface ItemIdStrategy {
+    fun getItemId(position: Int): Long
+}
```

Comment:
I partially agree. My concerns are mainly about function syntax readability which is no as good as just simple interface. Probably in future it would be much better thanks to function types. But I will change it anyway to functions ;)